tests/monitoring/metrics.go

```diff
@@ -100,7 +100,7 @@ var _ = Describe("[sig-monitoring]Metrics", decorators.SigMonitoring, func() {
 			err = virtapi.SetupMetrics()
 			Expect(err).ToNot(HaveOccurred())
 
-			err = virtcontroller.SetupMetrics(nil, nil, nil, nil, nil, nil, nil)
+			err = virtcontroller.SetupMetrics(nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil)
```

Comment:
It wasn't pretty before, now it certainly does not look better.

Reply:
Agreed, just pass `app`?